{"trial_id": "01", "hyperparameters": {"space": [{"class_name": "Int", "config": {"name": "neurons", "default": null, "conditions": [], "min_value": 1, "max_value": 100, "step": 1, "sampling": "linear"}}], "values": {"neurons": 51}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"/home/rpinosio/miniconda3/envs/ddb_tutorial/lib/python3.12/site-packages/keras_tuner/src/engine/base_tuner.py\", line 274, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"/home/rpinosio/miniconda3/envs/ddb_tutorial/lib/python3.12/site-packages/keras_tuner/src/engine/base_tuner.py\", line 265, in _run_and_update_trial\n    tuner_utils.convert_to_metrics_dict(\n  File \"/home/rpinosio/miniconda3/envs/ddb_tutorial/lib/python3.12/site-packages/keras_tuner/src/engine/tuner_utils.py\", line 132, in convert_to_metrics_dict\n    [convert_to_metrics_dict(elem, objective) for elem in results]\n     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/rpinosio/miniconda3/envs/ddb_tutorial/lib/python3.12/site-packages/keras_tuner/src/engine/tuner_utils.py\", line 145, in convert_to_metrics_dict\n    best_value, _ = _get_best_value_and_best_epoch_from_history(\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/rpinosio/miniconda3/envs/ddb_tutorial/lib/python3.12/site-packages/keras_tuner/src/engine/tuner_utils.py\", line 116, in _get_best_value_and_best_epoch_from_history\n    objective_value = objective.get_value(metrics)\n                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/rpinosio/miniconda3/envs/ddb_tutorial/lib/python3.12/site-packages/keras_tuner/src/engine/objective.py\", line 59, in get_value\n    return logs[self.name]\n           ~~~~^^^^^^^^^^^\nKeyError: 'val_accuracy'\n"}